src/core/include/openvino/core/preprocess/resize_algorithm.hpp

```diff
@@ -7,7 +7,13 @@
 namespace ov {
 namespace preprocess {
 
-enum class ResizeAlgorithm { RESIZE_LINEAR, RESIZE_CUBIC, RESIZE_NEAREST };
+enum class ResizeAlgorithm {
+    RESIZE_LINEAR,
```

Comment:
Can we add doxyden documentation like here: src/core/include/openvino/core/preprocess/color_format.hpp ?